apps/openmw/mwscript/guiextensions.cpp

```diff
@@ -189,15 +189,23 @@ namespace MWScript
                 }
                 else
                 {
-                    MWGui::GuiWindow gw = MWGui::GW_None;
-                    if (arg == 3)
+                    MWGui::GuiWindow gw;
+                    switch (arg) {
```

Comment:
Usually we place brackets to the separate line.

Reply:
Right.